cpp/src/parquet/CMakeLists.txt

```diff
@@ -407,3 +407,7 @@ endif()
 if(ARROW_WITH_ZSTD)
   add_definitions(-DARROW_WITH_ZSTD)
 endif()
+
+if(ARROW_CSV)
```

Comment:
this seems out of place?

Reply:
Do you means that I should add the code in function [ADD_PARQUET_TEST](https://github.com/apache/arrow/blob/master/cpp/src/parquet/CMakeLists.txt#L34)? Something like:
```
  if(REL_TEST_NAME STREQUAL "arrow-test" AND ARROW_CSV)
    add_definitions(-DARROW_CSV)
  endif()
```
This one also works :)